rules/stylistic-issues.js

```diff
@@ -21,28 +21,28 @@ module.exports = {
     'no-multiple-empty-lines': 2,
     'no-nested-ternary': 2,
     'no-new-object': 2,
-    'no-space-before-semi': 2,
+    'semi-spacing': 2,
     'no-spaced-func': 2,
     'no-ternary': 0,
     'no-trailing-spaces': 2,
     'no-underscore-dangle': 2,
-    'no-wrap-func': 2,
+    'no-extra-parens': 2,
```

Comment:
@easternbloc this is the rule that changed - we can configure it to only no-wrap-function as before